src/app/docs/split-view/split-view-docs.component.html

```diff
@@ -0,0 +1,233 @@
+<sky-docs-demo-page
+  moduleName="SkySplitViewModule"
+  moduleSourceCodePath="src/app/public/modules/split-view/"
+  pageTitle="Split view"
+>
+  <sky-docs-demo-page-summary>
+    Sample description.
+  </sky-docs-demo-page-summary>
+
+  <sky-docs-demo>
+    See documentation for how to setup demos.
+  </sky-docs-demo>
+
+  <sky-docs-design-guidelines>
+
+    <sky-docs-demo-page-section
+      heading="Usage"
+    >
+      <sky-docs-design-guideline
+        heading="Use when"
+        headingStyle="success"
+      >
+        <p>
+          Use split views when users need to work through a list of items with the intent of taking action on each item.
+        </p>
+        <sky-docs-design-guideline-thumbnail
+          caption="Do use split views when users take the same action for each item in the list."
+          captionType="success"
+          imageSource="~/assets/img/option-list-type-repeater.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+
+      <sky-docs-design-guideline
+        heading="Don't use when"
+        headingStyle="danger"
+      >
+        <p>
+          Don't use split views when users are likely to interact with a small number of items in the list. Use regular lists instead, and follow the <a href="https://developer.blackbaud.com/skyux/components/flyout#ux-guidelines">flyout guidelines</a> to determine whether to use flyouts or navigate to selected records.
```

Comment:
Should we update this guidance to recommend data managers instead of regular lists?

Reply:
@Blackbaud-SteveBrush, is this something to look into? Should I file an issue or post a question to the team Slack channel? Let me know if you think this is worth addressing before we merge these JSDoc comments.

Reply:
Let's keep it as-is for now. A new issue would be a good idea. Thanks!

Reply:
I created https://github.com/blackbaud/skyux-split-view/issues/38 to capture this.